Microsoft.DotNet.Interactive/ISupportNuget.cs

```diff
@@ -7,17 +7,12 @@
 
 namespace Microsoft.DotNet.Interactive
 {
-    public interface ISupportNuget
+    public interface ISupportNuget : IPackageRestoreContext
     {
-        // Set assemblyProbingPaths, nativeProbingRoots for Kernel.
-        // These values are functions that return the list of discovered assemblies, and package roots
-        // They are used by the dependecymanager for Assembly and Native dll resolving
-        public void Initialize(AssemblyResolutionProbe assemblyProbingPaths, NativeResolutionProbe nativeProbingRoots);
+        // KernelSupportsNugetExtension relies on access to the full PackageRestoreContextClass for state mutation
+        PackageRestoreContext PackageRestoreContext { get; }
```

Comment:
> PackageRestoreContext PackageRestoreContext { get; } [](start = 7, length = 53)

Is it intentional that `ISupportNuget` both HAS and IS an `IPackagRestoreContext`?